README.md

```diff
@@ -44,3 +44,32 @@ These are the arguments that apply to all commands:
     ```bash
     odf --context=$(oc config current-context) [commands]
     ```
+
+## Installation
+
+### Build from source
```

Comment:
Empty section, no need for title?
```suggestion
```

Reply:
This should refer to building from the github repository, if I'm not wrong. So it should be the title of all the points mentioned below.